src/haddock/modules/analysis/caprieval/capri.py

```diff
@@ -21,7 +21,7 @@
 from haddock.libs.libpdb import split_by_chain
 
 
-RES_TO_BE_IGNORED = ["SHA"]
+RES_TO_BE_IGNORED = ["SHA", "WAT"]
```

Comment:
I think it is fine for now.
In the future, once #144 is finalized, we can add the residues and solvents from those lists
https://github.com/haddocking/haddock3/blob/45eb54693d1d0ce495f69bc4921a9f1a4242c1af/src/haddock/core/supported_molecules.py